oauth2_http/java/com/google/auth/http/HttpCredentialsAdapter.java

```diff
@@ -11,11 +14,17 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 /**
  * A wrapper for using Credentials with the Google API Client Libraries for Java with Http.
  */
-public class HttpCredentialsAdapter implements HttpRequestInitializer {
+public class HttpCredentialsAdapter
```

Comment:
The prevailing style convention is to to wrap on as few lines as possible, so consider wrapping in 2 lines instead of 3.